.travis.yml

```diff
@@ -11,6 +11,7 @@ before_script:
   - sudo pkgutil --forget com.apple.pkg.CLTools_Executables
 
 script:
+  - ./install --force-curl
```

Comment:
Think it's probably worth `uninstall -f`ing before this as otherwise we're not testing a clean install.